toolkit/docs/building/dedicated_guides/build_images.md

```diff
@@ -0,0 +1,139 @@
+
+However, sometimes building quickly in any repo is valuable. The next section will show how to use the core repo in a similar way to what is described in the [CBL-MarinerTutorials](https://github.com/microsoft/CBL-MarinerTutorials) repo.
+
+## **Just build me an image, I don't care about anything else**
+
+Sometimes you just need to build an image quickly, and you don't want to fuss around with additional repos or directories. The following commands use flags that will make the core repo behave in a very similar manner to the [CBL-MarinerTutorials](https://github.com/microsoft/CBL-MarinerTutorials).
+
+Prepare your system by [installing the necessary prerequisites here](../prerequisites.md).
+
+```bash
+# Clone the CBL-Mariner repo
+git clone https://github.com/microsoft/CBL-Mariner.git
+cd CBL-Mariner
+
+# Sync to the latest stable build
+git checkout 2.0-stable
+cd ./toolkit
+sudo make clean
+
+# Build the full iso without looking at any local packages. REBUILD_PACKAGES=n fully
+#   disables all of the package building mechanisms.
+sudo make iso CONFIG_FILE=./imageconfigs/full.json REBUILD_PACKAGES=n REBUILD_TOOLS=y
+# --- or ---
+sudo make image CONFIG_FILE=./imageconfigs/core-efi.json REBUILD_PACKAGES=n REBUILD_TOOLS=y
+```
+
+## **I need to Validate Changes with Local Packages**
+
+The following directions will build both packages and images locally.
+
+### **1. Install Prerequisites**
+
+Prepare your system by [installing the necessary prerequisites here](../prerequisites.md).
+
+### **2. Picking a Branch or Tag**
+
+Please read [Clone and Sync to Stable Commit](../building.md#clone-and-sync-to-stable-commit) for details on picking a branch to develop from.
+
+When working with images, the `stable` approach is often the better choice but will lack bleeding edge packages. Generally when working with images, the version of a packages is less important than simply having the package available. **While much less likely with image work, still be aware that the stable branches trail behind the development branches and this can result in merge conflicts when it is time to create a PR.**
+
+The `hybrid` option minimizes the chance of package conflicts, but runs the risk of seeing unresolvable build cycles appearing.
+
+> Unlike when working on package changes, image changes are generally self-contained enough that a build with a `hybrid` or `stable` setup is sufficient for validation.
+
+### **3. Get a Toolchain**
+
+If you are using the `stable` or `hybrid` branch strategy, the toolchain will be handled automatically. Just invoke the `sudo make image ...` or `sudo make iso ...` commands as needed and the tooling will deal with the rest.
```

Comment:
```suggestion
If you are using the `stable` or `hybrid` branch strategy, the toolchain will be handled automatically. Invoke the `sudo make image ...` or `sudo make iso ...` commands as needed and the tooling will handle the rest.
```

Reply:
Are there full commands we should list here?

Reply:
No, the user will only need to run commands if they follow the link for development branches.